tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,89 @@
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use File::Basename;
+
+sub run {
+    my ($self, $args) = @_;
+    #Download unpublished DMS package and install rpm
+    my $ins_rke2 = get_var('RKE2_URL');
+    my $ins_kubectl = get_var('KUBECTL_URL');
+    my $ins_helm = get_var('HELM_URL');
+    #my $add_repo = get_var('HELM_REPO');
+   
+
+    #Create aws instance
+    my $provider;
+    my $instance;
+    select_host_console();
+
+    my $aws_test = get_var('PUBLIC_CLOUD_AWS_TEST', 'basic_sanity');
+    if ($aws_test) {
+        $instance = $self->{my_instance} = $args->{my_instance};
+        $provider = $self->{provider} = $args->{my_provider};    # required for cleanup
+    } else {
+        $provider = $self->{provider} = $self->provider_factory();    # required for cleanup
+        $instance = $self->{my_instance} = $provider->create_instance(check_guestregister => is_openstack ? 0 : 1);
+    }
+
+    registercloudguest($instance) if is_byos();
+    
+    #Update and install dependencies
+    $instance->ssh_assert_script_run('sudo zypper ref; sudo zypper -n up',
+        timeout => 1000);
+    record_info('Refresh and update');
+    sleep 90;    # wait for a bit for zypper to be available
+
+    #Install Docker (required for RKE2)
+
+    $instance->ssh_assert_script_run('sudo transactional-update -n pkg install -y curl',
+        timeout => 100);
```

Comment:
Please remember that tr-up changes need for reboot to be applied. Anyway being an heavy action, better to check if package eventually already present, you can skip it.
Anyway, we already have `lib/transactional.pm::trup_install()` doing that work.

Reply:
Should I replace it like this
`#Install Docker (required for RKE2)

    record_info 'Install tar', 'Install package curl using transactional server and reboot';
    trup_install "curl";
    record_info 'Install tar', 'Install package docker using transactional server and reboot';
    trup_install "docker";
    $instance->ssh_assert_script_run('sudo systemctl start docker',
        timeout => 100);
    $instance->ssh_assert_script_run('sudo systemctl enable docker',
        timeout => 100);
    $instance->ssh_assert_script_run('sudo systemctl status docker',
        timeout => 100);
    record_info('Installed Docker');
`

Reply:
Fixed